components/butterfly/src/member.rs

```diff
@@ -203,6 +203,21 @@ pub struct Membership {
     pub health: Health,
 }
 
+impl fmt::Display for Membership {
```

Comment:
My primary target for this (at least right now) is human readability, so that's what I optimized for.  If it turns out there's a need for downstream processing, I could see adding JSON output e.g. via a `-j` flag or some such would be a great addition.

Reply:
I think that if the RST file were initially in JSON (or some other commonly-used and easily-manipulated format) then this tool wouldn't need to exist, because people could take that format and be able to slice-and-dice it with whatever other tooling they wanted.

Since it's already so easy to convert it to JSON, that seems like a good common format to expose, since it doesn't block users from using other tools to process it. While this compact format is nice from a human-readability standpoint, it would require users to write their own custom code to do further analysis of it, rather than using common tools like `jq`.